domenetjenester/kompletthet/src/main/java/no/nav/foreldrepenger/kompletthet/impl/KompletthetsjekkerImpl.java

```diff
@@ -82,82 +83,56 @@ public KompletthetsjekkerImpl(BehandlingRepositoryProvider provider,
 
     @Override
     public KompletthetResultat vurderForsendelseKomplett(BehandlingReferanse ref, Skjæringstidspunkt stp) {
-        if (FagsakYtelseType.ENGANGSTØNAD.equals(ref.fagsakYtelseType())) {
-            if (utledAlleManglendeVedleggForForsendelse(ref).isEmpty()) {
+        if (ref.erRevurdering()) {
+            var behandling = behandlingRepository.hentBehandling(ref.behandlingId());
+            if (SpesialBehandling.skalGrunnlagBeholdes(behandling)) {
                 return KompletthetResultat.oppfylt();
             }
-            var ønsketFrist = LocalDateTime.now().plus(AUTO_VENTER_PÅ_KOMPLETT_SØKNAD.getFristPeriod());
```

Comment:
Er det noe spesiell grunn til at ventefristen er satt til dagens dato + 4 uker for ES ved manglende vedlegg?
FP/SVP har ventefrist mottattdato for søknad + 1 uke?

Kan man forenkle slik at FP/SVP/ES alle har samme frist?

Reply:
@espenwaaga bør gå helt greit. I praksis er det bare FP førstegang/endring som skal mangle vedlegg.
Fint med filter på !LASTET_OPP - så du får send senere mfl.
Mulig fristen skal opp litt når selvbetjening har funnet ut av varsling ...

Reply:
Som vi snakket om så beholder jeg filtere som det er i dag slik at det fungere som en guard mot vedlegg som er merkere LASTET_OPP ikke finnes i dokarkiv.